### PR TITLE
Add missing tsa config file

### PR DESCRIPTION
**Pull request by MichaelSimons**
Related to https://github.com/dotnet/dotnet-docker/issues/5967

### CODE REVIEW

File: .config/tsaconfig.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\NET Fundamentals\\.NET Acquisition\\Docker",
+    "iterationPath": "DevDiv",
+    "notificationAliases": [ "dotnetADTeam@microsoft.com" ],
+    "repositoryName": "dotnet-buildtools-prereqs-docker",
```
Comment (mthalman) [inline .config/tsaconfig.json:8]: This is the actual repo name in AzDO:

```suggestion
    "repositoryName": "dotnet-dotnet-buildtools-prereqs-docker",
```

Comment (MichaelSimons) [reply .config/tsaconfig.json:8]: The repo name don't have to match AzDO from other config files I have seen ([example](https://github.com/dotnet/arcade/blob/main/.config/tsaoptions.json#L8)).  The documentation doesn't call out how this is used.

Comment (MichaelSimons) [reply .config/tsaconfig.json:8]: Given the majority of repos are using the GH repo name, I decided to use it versus the AzDO repo name.

**Review by mthalman — APPROVED** [review]

**Comment by mthalman** [comment]
@MichaelSimons - The TSA error is still occurring after this merge: https://dev.azure.com/dnceng/internal/_build/results?buildId=2562948&view=results (internal link).

**Pull request merged**